client/fingerprint_manager.go

```diff
@@ -361,6 +348,21 @@ func (fm *FingerprintManager) fingerprintDriver(name string, f fingerprint.Finge
 		fm.nodeLock.Unlock()
 	}
 
```

Comment:
is there an existing unit test that can be modified to verify that an undetected driver will also always be unhealthy

Reply:
Good point, thankfully the unit tests in `fingerprint_manager_test` assert this (the periodic check is run very quickly and with a certain number of retries for unit tests to avoid race conditions).